docs/cli.md

```diff
@@ -94,10 +98,79 @@ With `--all` flag, you create config file including all default setting of offic
       executable_methods:
         - method_name: "node_governance_alarm"`
 ```
-
 Also, You can set the home directory of VATZ (default "~/.vatz") with `--home` flag.
 And you can create the new config file with your disired name with `--output` flag
 
+## Start
+You can start VATZ when you finish set up config default.yaml or your own config yaml file.
+```
+./vatz start --help                 
+start VATZ
+
+Usage:
+   start [flags]
+
+Flags:
+      --config string       VATZ config file. (default "default.yaml")
+  -h, --help                help for start
+      --log string          log file export to.
+      --prometheus string   prometheus port number. (default "18080")
+
+Global Flags:
+      --debug   Enable debug mode on Log.
+      --trace   Enable trace mode on Log.
+
+```
+You can simply start VATZ with command
+```
+~$ ./vatz start 
+```
+This will start VATZ with default config, that's been created from previous command `vatz init`. <br>
```

Comment:
This will start VATZ with the default config created from the previous `vatz init` command. You can set the exact config file with the `--config` flag, such as: